bin/do_deltas.py

```diff
@@ -170,10 +176,9 @@ def cont_fit(data):
     nit = args.nit
 
     log = open(args.log,'w')
-    data,ndata = io.read_data(args.in_dir,args.drq,args.mode,\
-                              zmin=args.zqso_min,zmax=args.zqso_max,nspec=args.nspec,log=log,keep_bal=args.keep_bal,bi_max = args.bi_max,order=args.order)
-    
-
+    data, ndata = io.read_data(args.in_dir,args.drq,args.mode,\
+                              zmin=args.zqso_min,zmax=args.zqso_max,nspec=args.nspec,log=log,keep_bal=args.keep_bal,bi_max = args.bi_max,order=args.order, best_obs = args.best_obs, single_exp = args.single_exp)
```

Comment:
Could you split this line on more lines?

Reply:
sure

Reply:
done